documentation/cloud-manager-guide/environments/backup-and-restore/restore-backup.md

```diff
@@ -0,0 +1,60 @@
+---
+keywords:
+title: Restore an Environment from a Backup
+description: How to restore a previous environment backup
+---
+# Restore an Environment from a Backup
+
+This guide provides an overview of the steps required to restore an environment to a previously backed up environment state.
+
+## Getting started
+
+To begin restoring an environment from a backup version, locate the backup from the list and select the corresponding **Options** (**...**) menu.
+
+From the **Options** menu, select **Restore** to open the *Restore Backup* dialog.
+
+![image description](images/restore-button.png)
+
+## Restore backup
+
+The *Restore Backup* dialog displays a warning notifying you that restoring a previous backup will erase your existing configuration.
+
+> **Note:** Ensure you have created a back up of your current environment configuration before proceeding.
```

Comment:
"backup" as one word for consistency. Might also want to link to the doc for creating a backup here in case they need it.